main.js

```diff
@@ -0,0 +1,275 @@
+/******CONSTANTS*******/
+const Medium = document.getElementById("Medium");
+const Hard = document.getElementById("Hard");
+const Easy = document.getElementById("Easy");
+const highScore = document.getElementById("high-score");
+const Score = document.getElementById("score");
+const speedRadios = document.querySelectorAll('input[name="speed"]');
+
+const canvas = getCanvas();
+const ctx = canvas.getContext("2d");
+let newDirection = "right", oldDirection = "right";
+let v =0;// 
+let food = {x:300,y:300}
+let highscore=0;
+
+let body = [{ x: 300, y: 300, newDirection: "right"}, { x: 270, y: 300, newDirection: "right"}, { x: 240, y: 300, newDirection: "right"}, { x: 210, y: 300, newDirection: "right"}];
```

Comment:
You should create a function that dynamically creates the initial state of the snake.
```ts
function createSnake(length: number, direction: 'right' | 'left' | 'up' | 'down'): Array<Snake> {
    ....
}
const snakeBody = createSnake(4, 'right');
```